readme.md

```diff
@@ -156,7 +156,7 @@ whereInResult.sql // => whereInQuery
 whereInResult.values // => [ [ 'fancy', 'boring' ] ]
 ```
 
-Put another way, calling `q.select('column1, column2')` is just as acceptable as calling `q.select('column1', 'column2')` and you should use whichever you prefer.
+All `value`s are automatically parameterized.  Put another way, calling `q.select('column1, column2')` is just as acceptable as calling `q.select('column1', 'column2')` and you should use whichever you prefer.
```

Comment:
"parameterized" isn't terribly clear but I meant to be talking about the arguments I called `value` in the functions above, not the `expression` arguments I talked about in this paragraph for something like `select`.